.env

```diff
@@ -1,2 +1,2 @@
-SALEOR_API_URL=https://latest.staging.saleor.cloud/graphql/
-CHECKOUT_APP_URL=http://localhost:3000
\ No newline at end of file
+SALEOR_API_URL=https://master.staging.saleor.cloud/graphql/
```

Comment:
Intentional?

Reply:
yes 😅

Reply:
@2can is this the variable we advise to use? I think it should be just the host no?

Reply:
@timuric at this point it's correct